Shared/XML/CXMLImpl.cpp

```diff
@@ -46,7 +46,8 @@ CXMLNode* CXMLImpl::ParseString(const char* strXmlContent)
     TiXmlDocument* xmlDoc = new TiXmlDocument();
     if (xmlDoc)
     {   
-        if (xmlDoc->Parse(strXmlContent, 0, TIXML_ENCODING_UTF8))
+        xmlDoc->Parse(strXmlContent, 0, TIXML_ENCODING_UNKNOWN);
```

Comment:
Actually the change from `UTF8` to `UNKNOWN` is not necessary. But I suppose our scripters will appreciate us not assuming the input is UTF8.

Reply:
Our file implementation uses `TIXML_DEFAULT_ENCODING`

Reply:
wtf is this garbage
![image](https://user-images.githubusercontent.com/923242/79033500-9c94b380-7ba6-11ea-855b-e6c76968b664.png)